config-files/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### PR Checklist
+
+- [ ] JIRA Ticket(s) included in PR Title and are formatted correctly if applicable (JIRA-123)
```

Comment:
Exactly why I requested commentary in the description. The entire line can be removed in that case, but we'd need to express that as an option. How shall we word that?